src/support/import-supervisor.js

```diff
@@ -98,6 +98,21 @@ function ImportSupervisor(services, config) {
     return dataAccess.createNewImportJob(newJob);
   }
 
+  /**
+   * Get all import jobs between the specified start and end dates.
+   * @param {string} startDate - The start date of the range.
+   * @param {string} endDate - The end date of the range.
+   * @param {string} importApiKey - The API
+   * @returns {Promise<Array<ImportJobDto>>}
+   */
+  async function getImportJobsByDateRange(startDate, endDate) {
+    const jobs = await dataAccess.getImportJobsByDateRange(startDate, endDate);
+    if (!jobs) {
+      throw new ErrorWithStatusCode('Not found', 404);
```

Comment:
On closer look this seems more like it would be caused by a unexpected service error, rather than a 404.